server/services/login.service.js

```diff
@@ -7,12 +8,27 @@ const loginService = async (email, password) => {
       throw new Error("Missing essential parameter(s)");
     }
     let user = await findUserByEmail(email);
+    let loginResponse;
+    if (!user) {
+      loginResponse = {
+        loginStatus: false,
+        msg: "User doesn't exist.",
+      };
+      return loginResponse;
+    }
+    if (!bcrypt.compareSync(password, user.passwd)) {
+      loginResponse = {
+        loginStatus: false,
+        msg: "Invalid password, please try again.",
+      };
```

Comment:
When logging in, instead of showing a specific message for username and password, it would be better to show a more general message to prevent attackers from probing for existing usernames.